cs/di-extensions.texy

```diff
@@ -5,66 +5,98 @@ DI: Rozšíření pro kontejner
 
 /---code neon
 extensions:
-	blog: MyBlogExtension
+	blog: BlogExtension
 \---
 
-Každé rozšíření `Compiler`u musí dědit od [Nette\DI\CompilerExtension|api:] a může implementovat tři různé metody, které jsou postupně volány, během sestavování `Container`u.
+Každé rozšíření `Compiler`u musí dědit od [Nette\DI\CompilerExtension|api:] a může implementovat tři různé metody, které jsou postupně volány, během sestavování `Container`u:
+
+1. loadConfiguration()
+2. beforeCompile()
+3. afterCompile()
 
 
 CompilerExtension::loadConfiguration() .{toc: loadConfiguration()}
```

Comment:
Asi i tady použít to `.[method]`, toc se pak udělá automaticky. A taky je možná zbytečné to `CompilerExtension::`